src/components/Category/Category.jsx

```diff
@@ -1,30 +1,32 @@
 import React from 'react';
 import {
   Box,
-  Flex,
   HStack,
   Text,
   MenuButton,
   MenuList,
   Menu,
   MenuItem,
   Button,
-  ButtonGroup,
-  IconButton,
+  Tabs, TabList, Tab,
 } from '@chakra-ui/react';
 import { Plus } from 'react-feather';
 
 const Category = () => {
   return (
-    <Box py="10" px="20" as="Category" fontSize={15} fontWeight={400}>
-      <HStack mx="50px" spacing={10} mb="10" color="black">
-        <Text>All</Text>
-        <Text> Furniture</Text>
-        <Text>Books</Text>
-        <Text>Toys</Text>
-        <Text>Medics</Text>
-        <Text>Appliances</Text>
-        <Text>Clothes</Text>
+    <Box py="10"  width={1080} as="Category" fontSize={15} fontWeight={400}>
+      <HStack spacing={10} mb="10" color="black">
+      <Tabs  variant="soft-rounded" colorScheme="gray">
+        <TabList>
+          <Tab>All</Tab>
+          <Tab>Furniture</Tab>
+          <Tab>Books</Tab>
+          <Tab>Toys</Tab>
+          <Tab>Medics</Tab>
+          <Tab>Appliances</Tab>
+          <Tab>Clothes</Tab>
```

Comment:
Could you please grab these categories form `categories.json` instead of having them hardcoded here?